src/tests/Equinor.ProCoSys.IPO.WebApi.Tests/Synchronization/BusReceiverServiceTests.cs

```diff
@@ -192,13 +192,14 @@ public async Task HandlingCommPkgTopicWithoutFailure()
         public async Task HandlingCommPkgTopic_Move_WithoutFailure()
         {
             var message = $"{{\"Plant\" : \"{plant}\", \"ProjectGuid\" : \"{_project2Guid}\", \"ProCoSysGuid\" :\"{s_commPkgGuid3Project2}\", \"Description\" : \"{description}\"}}";
+            _invitationRepository.Setup(x => x.IsExistingProject(_project2Guid)).Returns(true);

```

Comment:
Unit test is green even without these two setup lines, so I assume they can be removed?

Reply:
This is done to mark the condition properly (readability). By coincidence they will also run successfully without the condition setup. But that is related to the fact that the bool variables is set to true by default.